app/config.py

```diff
@@ -1,7 +1,20 @@
 import os
+import json
 
 basedir = os.path.abspath(os.path.dirname(__file__))
+configdir = os.path.join(basedir, 'config')
+overridedir = os.path.join(configdir, 'overrides')
 
+def get_json_dict(path):
+    try:
+        with open(path) as file:
+            return json.load(file)
+    except:
+        return dict()
+
+def make_merged_cfg(paths):
+    print(paths)
```

Comment:
Thanks for these changes! I'm excited. I think you left a `print` statement here unintentionally